conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -96,6 +104,51 @@ def __init__(self, conanfile, namespace=None, prefix="/"):
         self.apple_isysroot_flag = isysroot_flag
         self.apple_min_version_flag = min_flag
 
+    def _resolve_android_cross_compilation(self):
+        # Issue related: https://github.com/conan-io/conan/issues/13443
+        ret = {}
+        if not self._is_cross_building or not self._conanfile.settings.get_safe("os") == "Android":
+            return ret
+        ndk_path = self._conanfile.conf.get("tools.android:ndk_path", check_type=str)
+        if ndk_path:
```

Comment:
What if ``ndk_path`` is not defined? Isn't it mandatory in other places?

Reply:
It's true that if it's not defined, in other toolchains like Meson or CMake, it is raised as an exception, but do we want to do that here? It'd be a breaking change because people have always included all the needed flags via `buildenv`.